pkg/coord/adapter.go

```diff
@@ -292,10 +323,14 @@ func (a *adapter) ListDataspace(ctx context.Context) ([]*dataspaces.Dataspace, e
 	return nil, fmt.Errorf("ListDataspace not implemented")
 }
 
-func (a *adapter) AddDataspace(ctx context.Context, ks *dataspaces.Dataspace) error {
+func (a *adapter) AddDataspace(ctx context.Context, ds *dataspaces.Dataspace) error {
 	return fmt.Errorf("addDataspace not implemented")
```

Comment:
Well, lets implement it in next PR then!

Reply:
Implemented these methods